test/log-generators/cronjobs/kaconf.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: kubearchive.kubearchive.org/v1alpha1
+kind: KubeArchiveConfig
+metadata:
+  name: kubearchive
+spec:
+  resources:
+    - selector:
+        apiVersion: batch/v1
+        kind: Job
+      archiveWhen: has(status.startTime)
+      deleteWhen: has(status.completionTime)
```

Comment:
Should this be part of this PR?

Reply:
Removed and now included in https://github.com/kubearchive/kubearchive/pull/431.